examples/pytorch/requirements.txt

```diff
@@ -6,4 +6,4 @@ ray-on-aml==0.2.4
 ray[train]==2.4.0
 torch-geometric==2.0.4
 torch-scatter==2.0.9
-torch-sparse==0.6.13
+torch-sparse==0.6.16
```

Comment:
Make these versions the same as the ones in requirements.txt, otherwise we can maybe remove this file since we are using bazel for everything now.

Reply:
Yeah, this file is not needed. Thanks!